guides/gatherers.md

```diff
@@ -0,0 +1,69 @@
+# Gatherers
+
+- [Introduction](#introduction)
+- [Available Gatherers](#available-gatherers)
+    - [Corosync](#corosync)
+
+## Introduction
+
+Gatherers can be thought of as functions:
+- they have a name
+- they accept argument(s)
+- they return a value, the gathered [Fact](./specification.md#facts)
+
+Facts Gathering process in a nutshell 
+```
+fact = gatherer(argument)
+```
+
+## Available Gatherers
+
+Here's a collection of build-in gatherers, with information about how to use them.
+
+| name                          | implementation          
+| ----------------------------- | ------------------------------------------
+| [`corosync.conf`](#corosync)  | [trento-project/agent/../gatherers/corosyncconf.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/corosyncconf.go)
+
+### Corosync
```

Comment:
Let's use `corosync.conf` here. We have other gatherers about corosync (`corosynccmapctl` which returns runtime corosync configuration values)